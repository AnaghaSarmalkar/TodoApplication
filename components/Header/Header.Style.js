import { StyleSheet } from "react-native";

export const style = StyleSheet.create({
  header: {
    fontSize: 32,
    backgroundColor: "#fff",
  },
});
