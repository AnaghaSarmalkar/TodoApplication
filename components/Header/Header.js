import React from "react";
import { Text } from "react-native";
import { styles as basicStyles } from "./Header.Style";

export default function Header({ styles = basicStyles, title }) {
  return <Text style={styles.header}>{title}</Text>;
}
