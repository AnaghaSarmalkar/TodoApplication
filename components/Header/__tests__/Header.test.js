import { render } from "@testing-library/react-native";
import React from "react";
import Header from "../Header";

describe("", () => {
  it("renders Header component", async () => {
    const headerTitle = "Test Header Title";
    const { toJSON } = render(<Header title={headerTitle} />);

    expect(toJSON()).toMatchSnapshot();
  });
});
