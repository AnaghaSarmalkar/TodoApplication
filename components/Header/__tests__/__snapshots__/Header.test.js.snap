// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` renders Header component 1`] = `
<View>
  <Text
    style={
      {
        "backgroundColor": "#fff",
        "fontSize": 32,
      }
    }
  >
    Test Header Title
  </Text>
</View>
`;
